_episodes/03-sql-joins.md

```diff
@@ -21,6 +21,17 @@ keypoints:
 To combine data from two tables we use the SQL `JOIN` command, which comes after
 the `FROM` command.
 
+Earlier, we talked about relational databases.
+
+Database tables are used to organize and group data by common characteristics or principles.  
+Often, we need to combine elements from separate tables into a single tables or queries for analysis and visualization.
+A JOIN is a means for combining columns from one (self-join) or more tables by using values common to each.
+
+The JOIN command combined with ON is used to combine fields from separate tables.  
+
+An SQL join clause combines columns from one or more tables in a relational database. 
+It creates a set that can be saved as a table.
```

Comment:
Saving the results of the query as a new table is not that common and goes against normalization. Or do you mean a file, e.g. CSV?